Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -819,7 +872,33 @@ void OnServerErrorInternal(NetworkConnection conn, ErrorMessage msg)
 
         #region Client Internal Message Handlers
 
-        void OnClientConnectInternal(NetworkConnection conn, ConnectMessage message)
+        void OnClientAuthenticateInternal(NetworkConnection conn, ConnectMessage message)
```

Comment:
Rename to OnClientConnectInternal

Reply:
There already is a OnClientConnectInternal down below a bit

Reply:
yes,   please get rid of it.

Move all the code from OnClientConnectInternal into OnClientAuthenticated